### PR TITLE
for phalcon-phpstorm

### PR DESCRIPTION
**Pull request by niboge**
**Reasons for making this change:**

_TODO_

**Links to documentation supporting these rule changes:** 

_TODO_

If this is a new template: 
- **Link to application or project’s homepage**: _TODO_

### CODE REVIEW

**Comment by shiftkey** [comment]
@niboge I need some more information on why this needs to be it's own template, versus incorporating this into the existing PHP template...

File: phalconPhpStorm.gitignore [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Logs
+*.log.*
+*.log.wf
+*.docx
```
Comment (sergeyklay) [inline phalconPhpStorm.gitignore]: `docx` does not related to the Phalcon

File: phalconPhpStorm.gitignore [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Logs
+*.log.*
+*.log.wf
```
Comment (sergeyklay) [inline phalconPhpStorm.gitignore]: `log.wf` does not related to the Phalcon

File: phalconPhpStorm.gitignore [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+*.docx
+*.wf
```
Comment (sergeyklay) [inline phalconPhpStorm.gitignore]: `wf` does not related to the Phalcon

File: phalconPhpStorm.gitignore [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+logfile
```
Comment (sergeyklay) [inline phalconPhpStorm.gitignore]: `logfile ` does not related to the Phalcon

File: phalconPhpStorm.gitignore [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+# cache
+apps/var/cache/*
+apps/var/temp
+apps/cli/tools/tmp
+
+# slef config
+apps/library/Smarty/config
+apps/library/Smarty/config/www/map.json
+apps/library/Smarty/config/admin/map.json
```
Comment (sergeyklay) [inline phalconPhpStorm.gitignore]: `Smarty` does not related to the Phalcon

File: phalconPhpStorm.gitignore [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+apps/config/config-staging.php
+robots.txt
+sftp-config.json
+
+# ignore front_end view
+apps/modules/www/views
+apps/modules/admin/views
+apps/config/env_para.php
```
Comment (sergeyklay) [inline phalconPhpStorm.gitignore]: `env_para` does not related to the Phalcon

File: phalconPhpStorm.gitignore [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+public/static
+public_admin/static
```
Comment (sergeyklay) [inline phalconPhpStorm.gitignore]: `public_admin` does not related to the Phalcon

**Comment by shiftkey** [comment]
Closing this out due to inactivity

**Pull request closed without merging**